### DISCUSSION TITLE
How to deploy our website to www web for free using github?

### DISCUSSION THREAD

**Discussion by DanMathew7** [AlanBinu007/Realtime-Chat-Application-ReactJs#16] [Q&A] [answered]
How to deploy our website to www web for free using github?

**Comment by AlanBinu007** [comment, marked as answer]
https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site

**Reply by DanMathew7** [reply]
Thanks, Alan
